cosmosdb/sql/ps-account-update.ps1

```diff
@@ -1,4 +1,4 @@
-# Update an Azure Cosmos Account and add a region
```

Comment:
There are two files in the azure-docs-pr repo that reference this file, we should remove one of them. 

\azure-docs-pr\articles\cosmos-db\scripts\powershell\sql\ps-account-update.md

\articles\cosmos-db\scripts\powershell\common\ps-account-update.md

Reply:
I will modify the path in the samples page once this PR is merged.